testing/trino-benchto-benchmarks/src/test/resources/statistics/tpcds/sf1000/README.txt

```diff
@@ -0,0 +1,2 @@
+These statistics are copied from the hive connector from TPCDS schema with sf1000.
+So there are no min nor max values for char-based columns.
```

Comment:
were they produced with Hive partitioned tables, or hive unpartitioned tables?

Reply:
worth documenting in the README?

Reply:
I think so. Will do that